test/packages/good/kibana/ml/good-ml-module-abc-1.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Would be nice to have some example content, mostly to see how it will look and as an example.

Reply:
I've added the NGINX module saved object in https://github.com/elastic/package-spec/pull/152/commits/05395ed375a6b34c261ce1611cb0c84442b94d29